spec/index.html

```diff
@@ -504,6 +504,43 @@ <h3>RDF Documents and Syntaxes</h3>
       effect on the convenience of working with the <a>RDF document</a>,
       they are not significant for its meaning.</p>
   </section>
+
+  <section id="section-version-announcement">
+    <h3>RDF Version Announcement</h3>
+
+    <p>To allow RDF parsers to error or warn about unsupported RDF versions as early as possible,
+      RDF serialization formats are expected to allow a version to be specified,
+      via either the HTTP Content-Type header,
+      a version announcement in a format-specific syntax,
```

Comment:
Related to [Pierre-Antoine's comment](https://github.com/w3c/rdf-concepts/pull/187/files#r2046873840) about "media-type parameter" in general:

Wouldn't the inclusion of the version in the syntax be the most reliable and only place that a parser should be concerned with? So, why bother with the version parameter elsewhere? Optimisation? Authoritative?

I stress again that if there are multiple places for a parser to look at as I've mentioned in https://github.com/w3c/rdf-concepts/pull/187#pullrequestreview-2771652770 , the parser should also be advised to pick the most authoritative one among possible options, and probably emit a warning when the version information differ.

Reply:
I sympathize with the concern of having multiple ways to convey the information, but both have their specific advantages:
- the format-specific version indicator is usable in contexts were media-type is not available (e.g. most file systems)
- the media-type parameter can be used during HTTP content-negotiation

Reply:
Ack. Then, definitely mention priority: https://github.com/w3c/rdf-concepts/pull/187/files#r2047503219